pkg/csi/service/common/constants.go

```diff
@@ -142,4 +142,8 @@ const (
 
 	// IopslimitMigrationParam is raw vSAN Policy Parameter
 	IopslimitMigrationParam = "iopslimit-migrationparam"
+
+	// AnnMigratedTo annotation is added to a PVC and PV that is supposed to be
```

Comment:
Can we remove mention of dynamic from comment. This applies to static PVs and PVCs as well.
I suggest following would be good.

> AnnMigratedTo annotation is added to a PVCs and PVs migrated to CSI Driver

Reply:
Done